src/dotnet/APIView/APIViewWeb/LeanModels/ReviewListModels.cs

```diff
@@ -96,15 +96,18 @@ public class BaseListitemModel
         public string Id { get; set; } = IdHelper.GenerateId();
         public string PackageName { get; set; }
         public string Language { get; set; }
-    }
-
+    }    
+    
     public class ReviewListItemModel : BaseListitemModel
     {
         public HashSet<string> Subscribers { get; set; } = new HashSet<string>();
         public List<ReviewChangeHistoryModel> ChangeHistory { get; set; } = new List<ReviewChangeHistoryModel>();
         public List<ReviewAssignmentModel> AssignedReviewers { get; set; } = new List<ReviewAssignmentModel>();
         public bool IsClosed { get; set; }
         public bool IsApproved { get; set; }
+        public bool IsNamespaceApproved { get; set; }
```

Comment:
This is not required. `IsApproved` property is currently used for review name approval so we don't need a new property.